include/tcp_handler.h

```diff
@@ -18,6 +18,10 @@
 #include "lwip/sys.h"
 #include <lwip/netdb.h>
 
+#ifdef __cplusplus
```

Comment:
Why is this needed here?

Reply:
yes we are not using it but if someone add this repo in their project, the compiler will know that this file is C not C++ and will compile it with C compiler instead of C++ which add random numbers at the end of variable name for polymorphism and linker will fail.